### DISCUSSION TITLE
Are you still updating your repo?

### DISCUSSION THREAD

**Discussion by WoodpeckerBaby** [qnblackcat/uYouPlus#864] [Q&A]
The uYou+ there is outdated.

**Comment by yodaluca23** [comment]
https://github.com/qnblackcat/AltStore/issues/9

**Reply by WoodpeckerBaby** [reply]
It's still one version behind. 

![image](https://user-images.githubusercontent.com/14303318/211183780-fa4c177d-fabe-427b-bf94-3bd3a6431686.png)
